client/Pages/EventTypeDetails/QueryTab.elm

```diff
@@ -45,6 +45,8 @@ queryTabHeader settings model query =
         , helpIcon "Nakadi SQL" queryHelp BottomRight
         , label [ class "query-tab__label" ] [ text " Status: " ]
         , span [ class statClass ] [ text "active" ]
+        , label [ class "query-tab__label" ] [ text " Envelope: " ]
+        , span [] [ text (boolToString query.envelope) ]
```

Comment:
You can add help tip here. Like here https://github.com/zalando-nakadi/nakadi-ui/blob/master/client/Pages/EventTypeDetails/View.elm#L479